### ISSUE TITLE
[Bug]Evaluation can produce NaN actions when normalization delta is zero

### ISSUE THREAD

**Issue by Papaercold** [X-Square-Robot/wall-x#113]
## Description

While fine-tuning and evaluating the VLA model with LeRobot data, I observed a case where training loss looked normal, but evaluation produced all-NaN action vectors.

I traced this to a normalization mismatch between the LeRobot training collator and the shared model normalizer.

The LeRobot training collator already guards against zero normalization deltas in:

```text
wall_x/data/backends/lerobot/loader.py:473-481
```

```python
delta = torch.where(delta == 0, torch.ones_like(delta), delta)
x = (action - min_stat) / delta
x = x * 2 - 1
x = torch.clamp(x, -1, 1)
```

This training-side normalization is used for both `agent_pos` and `action` before they are passed into the model.

However, the shared model normalizer previously divided directly by `self.delta[dataset_name]` in:

```text
wall_x/model/core/action/normalizer.py:284-295
```

For constant action/state dimensions, the normalization statistics can have:

```text
q99[i] == q01[i]
delta[i] == 0
```

which can lead to:

```text
0 / 0 -> NaN
```

Since `torch.clamp()` does not remove NaNs, these values can propagate and result in all-NaN action vectors during evaluation/inference.

## Expected Behavior

Training and evaluation/inference normalization should handle zero-delta dimensions consistently.

## Observed Behavior

Training remained stable, but evaluation produced all-NaN action vectors in my fine-tuning + evaluation workflow.

## Proposed Fix

Apply the same zero-delta guard in `Normalizer.normalize_data()` that is already used in the LeRobot training collator.

I opened a PR with the proposed fix: #112

**Comment by StarrickLiu** [comment]
Thanks for the precise root-cause analysis @Papaercold — you pinned it exactly: the LeRobot training collator guards `delta == 0`, but `Normalizer.normalize_data()` divided by `delta` directly, so constant dimensions (`q99 == q01` → `delta == 0`) gave `0 / 0 → NaN` at eval/inference while training looked fine.

Confirmed and fixed by #112, which applies the same zero-delta guard so training and eval normalization are consistent. Merging that now — this issue will close with it.
